src/main/java/slimeknights/tconstruct/plugin/crt/CRTHelper.java

```diff
@@ -0,0 +1,150 @@
+package slimeknights.tconstruct.plugin.crt;
+
+import com.blamejared.crafttweaker.api.CraftTweakerAPI;
+import com.blamejared.crafttweaker.impl.commands.CTCommandCollectionEvent;
+import com.blamejared.crafttweaker.impl.commands.script_examples.ExampleCollectionEvent;
+import com.blamejared.crafttweaker.impl.fluid.MCFluidStackMutable;
+import com.blamejared.crafttweaker.impl_native.item.ExpandItem;
+import com.google.gson.JsonElement;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.text.Color;
+import net.minecraft.util.text.StringTextComponent;
+import net.minecraft.util.text.TextFormatting;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fluids.FluidStack;
+import net.minecraftforge.registries.ForgeRegistries;
+import slimeknights.tconstruct.TConstruct;
+import slimeknights.tconstruct.library.MaterialRegistry;
+import slimeknights.tconstruct.library.TinkerRegistries;
+import slimeknights.tconstruct.library.materials.IMaterial;
+import slimeknights.tconstruct.library.materials.Material;
+import slimeknights.tconstruct.library.materials.MaterialId;
+import slimeknights.tconstruct.library.modifiers.Modifier;
+import slimeknights.tconstruct.library.modifiers.ModifierEntry;
+import slimeknights.tconstruct.library.modifiers.ModifierId;
+import slimeknights.tconstruct.library.tinkering.IMaterialItem;
+import slimeknights.tconstruct.library.tools.item.ToolCore;
+
+import java.util.stream.Collectors;
+
+public class CRTHelper {
+  
+  public static MaterialId getMaterialId(String materialId) {
```

Comment:
In general, I like javadocs on public methods without `@Override` annotations. You can skip it on event listeners, but on utilities just a simple javadoc explaining the method for consistency. On this method specifically, worth mentioning it validates the material ID.